src/include/daos_srv/ras.h

```diff
@@ -136,6 +140,18 @@ ds_notify_ras_event(ras_event_t id, char *msg, ras_type_t type, ras_sev_t sev,
 		    uuid_t *cont, daos_obj_id_t *objid, char *ctlop,
 		    char *data);
 
+/**
+ * A printf-style message-formatting wrapper for ds_notify_ras_event. If the
+ * resulting message is too long for DAOS_RAS_STR_FIELD_SIZE, it will be ended
+ * with a '$' to indicate so. See ds_notify_ras_event for parameter
+ * documentation.
+ */
+void
```

Comment:
Should add this attribute to this API so the compiler can help check the format string fro issues

void d_log(int flags, const char *fmt, ...) __attribute__ ((__format__(__printf__, 2, 3)));

I believe it's argument number of format string and first var arg

Reply:
Thanks. Will do, either in this or another patch, depending on how things go.